packages/sdk-react/src/components/providers/FusionAuthProvider.tsx

```diff
@@ -47,6 +47,8 @@ const FusionAuthProvider: FC<
     config.shouldAutoRefresh ?? false,
   );
 
+  const manageAccount = () => core.manageAccount();
```

Comment:
Can we please put this in a `useCallback` and move it to `useRedirecting`? Since it's similar to the methods in there.

Reply:
@JakeLo123 good call, updated!